owncloud/owncloud.py

```diff
@@ -651,6 +635,130 @@ def get_shares(self, path='', **kwargs):
                 return shares
         raise ResponseError(res)
 
+    def create_user(self, user_name, initial_password):
+        """Create a new user with an initial password via provisioning API.
+	   It is not an error, if the user already existed before.
+	   If you get back an error 999, then the provisioning API is not enabled.
+
+        :param user_name:  name of user to be created
+        :param initial_password:  password for user being created
+        :returns: True on success
+        :raises: ResponseError in case an HTTP error status was returned
+
+	"""
+        res = self.__make_ocs_request(
+                'POST',
+                self.OCS_SERVICE_CLOUD,
+                'users',
+                data = { 'password': initial_password, 'userid': user_name }
+                )
+
+	# We get 200 when the user was just created.
+        if res.status_code == 200:
+	    # We get an inner 102 although we have an outer 200 when the user already exists.
+            tree = ET.fromstring(res.text)
+            self.__check_ocs_status(tree, [100, 102])
+	    # return ET.tostring(tree)
+	    return True
+
+        raise ResponseError(res)
+
+    def delete_user(self, user_name):
+        """Deletes a user via provisioning API.
+	   If you get back an error 999, then the provisioning API is not enabled.
+
+        :param user_name:  name of user to be deleted
+        :returns: True on success
+        :raises: ResponseError in case an HTTP error status was returned
+
+	"""
+        res = self.__make_ocs_request(
+                'DELETE',
+                self.OCS_SERVICE_CLOUD,
+                'users/'+user_name
+                )
+
+	# We get 200 when the user was deleted.
+        if res.status_code == 200:
+	    return True
+
+        raise ResponseError(res)
+
+    def isUserCreated (self, user_name):
```

Comment:
=> `is_user_created` (seems to be PEP8 too https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)